src/components/shared/MountTransition.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { State, Transition, TransitionProps } from "react-spring";
+import { animationConfig } from "../../utils/animation";
+import { Omit } from "utility-types";
+
+interface MountTransitionProps
+  extends Omit<Omit<TransitionProps<boolean>, "items">, "children"> {
+  visible: boolean;
+  children: (
+    params: React.CSSProperties,
+    state: State,
+    index: number
+  ) => React.ReactNode;
+}
+
+const MountTransition: React.FunctionComponent<
+  MountTransitionProps
+> = props => {
+  return (
+    <Transition items={props.visible} {...animationConfig} {...props}>
```

Comment:
AFAIK items can be an array or undefined.
http://react-spring.surge.sh/transition#/red

Reply:
It can accept boolean too. See the 3rd example on that page.